.github/workflows/goreleaser.yml

```diff
@@ -62,14 +58,11 @@ jobs:
     - name: Package and push charts
       run: |
         chartVersion=$(echo ${{ github.ref_name }} | cut -c 2-)
-        make chart CONTROLLER_MANAGER_IMAGE=${{ env.REGISTRY }}/${{ env.CONTROLLER_MANAGER_IMAGE }}:${{github.ref_name}}
-        helm package ${{ env.CM_CHART_PATH }} --destination . --version ${chartVersion} --app-version ${chartVersion}
-        helm push ./controller-manager-chart*.tgz oci://${{ env.REGISTRY }}/kubestellar/kubestellar
-        
+
+        make core-chart IMAGE_TAG=${{chartVersion}}
```

Comment:
This caused the failure at https://github.com/kubestellar/kubestellar/actions/runs/11170183214 . The problem here is that the syntax `${{chartVersion}}` is asking for GitHub to expand `chartVersion` but that is not an environment version at the GitHub workflow level. It is a shell variable inside the shell script here. You want to use the same syntax seen elsewhere in the same script for expansion of that shell variable.